src/test/java/com/google/api/codegen/testdata/discoveries/ruby/ruby_compute.v1.json.baseline

```diff
@@ -29,9 +29,11 @@ items = service.fetch_all do |token|
   service.list_aggregated_addresses(project, page_token: token)
 end
 
-items.each do |name, addresses_scoped_list|
-  # TODO: Change code below to process each (name, addresses_scoped_list) pair:
-  puts String(name << " => " << addresses_scoped_list.to_json
+items.each do |item|
+  item.each do |name, addresses_scoped_list|
+    # TODO: Change code below to process each (name, addresses_scoped_list) pair:
+    puts String(name) + " => " + addresses_scoped_list.to_json
+  end
```

Comment:
Given the way the library currently handles maps, it seems to me that to be conceptually right (and avoid confusing the user), the variable names for this case will need to change---something like: `items` => `page`, `item` => `items`.